src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_NUM = 5; //FIGURE_NUM - number of figure subclasses

```

Comment:
It is recommended to use more descriptive constant names instead of providing comments. Consider renaming FIGURE_NUM to NUMBER_OF_FIGURES for better readability and omit comment.